h/views/account_signup.py

```diff
@@ -348,23 +353,57 @@ def is_authenticated(request):
     )
 
 
-def encode_idinfo_token(
+IDINFO_RFP_SESSIONKEY_FMT = "idinfo.rfp.{audience}"
+
+
+def encode_idinfo_token(  # noqa: PLR0913
```

Comment:
`# noqa: PLR0913` is `too-many-arguments` again. We should have a dataclass for the JWT payload, instead of having each payload field as a separate argument. This is part of a future refactoring that I have planned, so leaving it out of scope for this PR.